docs/security.rst

```diff
@@ -0,0 +1,2 @@
+.. include:: ../SECURITY.md
```

Comment:
`warnings-plugin/docs/index.rst:5: WARNING: toctree contains reference to document 'security' that doesn't have a title: no link will be generated`

Sphinx can't handle Markdown, but GitHub supports RST.

Reply:
Indeed: efac4e5

Reply:
Is there a way to align the file extension and still include it here?

Reply:
Contribution guide is included in same way. I think its now faiking for some changes in Sphinx conf.py